superset/models/sql_lab.py

```diff
@@ -110,11 +116,11 @@ class Query(
     sql_editor_id = Column(String(256), index=True)
     schema = Column(String(256))
     catalog = Column(String(256), nullable=True, default=None)
-    sql = Column(MediumText())
+    sql = Column(LongText())
```

Comment:
@john-bodley Just to confirm.. is `sql` `LongText` or `MediumText`? The reason I'm asking is because it's not being affected by your script.

Reply:
Per [here]([here](https://github.com/apache/superset/blob/f29e1e4c29a46f7d607cfa59adb8bb21d107091c/superset/migrations/versions/2019-05-06_14-30_afc69274c25a_alter_sql_column_data_type_in_query_mysql_table.py#L36-L46)) `sql` is `LongText` and isn't impacted by any of the aforementioned migrations.